4.cellprofiler_analysis/pipelines/analysis_4ch_with_sam.cppipe

```diff
@@ -48,31 +59,45 @@ NamesAndTypes:[module_num:3|svn_version:'Unknown'|variable_revision_number:8|sho
     Set intensity range from:Image metadata
     Maximum intensity:255.0
     Select the rule criteria:and (metadata does Channel "02")
-    Name to assign these images:488_1
+    Name to assign these images:CL_488-1
     Name to assign these objects:Cell
     Select the image type:Grayscale image
     Set intensity range from:Image metadata
     Maximum intensity:255.0
     Select the rule criteria:and (metadata does Channel "03")
-    Name to assign these images:488_2
+    Name to assign these images:CL_488_2
     Name to assign these objects:Cell
     Select the image type:Grayscale image
     Set intensity range from:Image metadata
     Maximum intensity:255.0
     Select the rule criteria:and (metadata does Channel "04")
-    Name to assign these images:561
+    Name to assign these images:CL_561
+    Name to assign these objects:Cell
+    Select the image type:Grayscale image
+    Set intensity range from:Image metadata
+    Maximum intensity:255.0
+    Select the rule criteria:and (file does endwith ".png")
+    Name to assign these images:mask
     Name to assign these objects:Cell
     Select the image type:Grayscale image
     Set intensity range from:Image metadata
     Maximum intensity:255.0
 
 Groups:[module_num:4|svn_version:'Unknown'|variable_revision_number:2|show_window:False|notes:['Given we are using timelapse data, we group by Well and FOV.']|batch_state:array([], dtype=uint8)|enabled:True|wants_pause:False]
-    Do you want to group your images?:Yes
+    Do you want to group your images?:No
     grouping metadata count:2
     Metadata category:Well
     Metadata category:FOV
 
-ImageMath:[module_num:5|svn_version:'Unknown'|variable_revision_number:5|show_window:False|notes:['This module was suggested by Saguaro Bio and approved by Jenna Tomkinson and added by Mike Lippincott. ', '', 'We want a more robust segmentation of the cytoplasm. To do so, we utilize the ImageMath module to add all the cytoplasmic channel pixel values together to merge into one channel for segmenting whole cells. This ensures the best segmentation across cell states where the zwitterionic dyes are drastically changing.']|batch_state:array([], dtype=uint8)|enabled:True|wants_pause:False]
+ConvertImageToObjects:[module_num:5|svn_version:'Unknown'|variable_revision_number:1|show_window:False|notes:[]|batch_state:array([], dtype=uint8)|enabled:True|wants_pause:False]
```

Comment:
Recommend in GUI to add comment here for what this module is doing, e.g., this module will load in the SAM2 Masks of nuclei to be used for feature extraction.

Reply:
applied!